### ISSUE TITLE
Color changed automatically after select

### ISSUE THREAD

**Issue by nazrulcse** [photopea/photopea#850]
![email 1](https://user-images.githubusercontent.com/1034949/44895506-747c7780-ad16-11e8-806c-9cabef6467e8.jpg)

After selecting this image from my computer colors of the image changed automatically like below:

![photopea online image editor](https://user-images.githubusercontent.com/1034949/44895560-af7eab00-ad16-11e8-8bfe-ff33d73dd055.png)

**Comment by photopea** [comment]
We use JPEG decoder from PDF.js .

We already had this problem in the past: https://github.com/photopea/photopea/issues/252 .  It was solved in the past by using some random modification of PDF.js JPEG decoder from some random person from the internet. But then, we updated the JPEG decoder by the latest one from official PDF.js and it is broken  again.

I don't want to maintain my own branch of JPEG decoder with dozens of patches I don't understand, so I reported it to PDF.js and will wait for their response. https://github.com/mozilla/pdf.js/issues/10025
